src/GitVersionTask/AssemblyInfoBuilder/AssemblyInfoBuilder.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+
 using GitVersion;
 
 public class AssemblyInfoBuilder
 {
     public string GetAssemblyInfoText(VersionVariables vars)
     {
+        var v = vars.ToArray();
```

Comment:
FYI, `.ToList()` is actually more efficient because an IEnumerable length is not fixed the array buffer has to be expanded multiple times, lists have that functionality built in. Not that it really matters in most code but just a FYI :)